src/overlays/actors/ovl_En_Ending_Hero6/z_en_ending_hero6.c

```diff
@@ -130,7 +165,7 @@ void EnEndingHero6_Draw(Actor* thisx, PlayState* play) {
     EnEndingHero6* this = THIS;
     s32 index = 0;
 
-    if (this->isIdle == 1) {
+    if (this->isIdle == true) {
```

Comment:
Not really a big deal, but based on this I wonder if `isInit` would be better for this (would think it applies to the other ending heros but we have less info on them?

Reply:
Yeah, we have very little to go off of. It is technically set to true in `EnEndingHero6_SetupIdle`, so if `EnEndingHero6_Init` were to setup a different action func (not that there's any other ones), then `isIdle` wouldn't be set to true